packages/api/cms-api/src/generator/generate-crud-input.ts

```diff
@@ -194,7 +194,7 @@ export async function generateCrudInput(
                     decorators.push("@IsString({ each: true })");
                 } else if (refType == "integer") {
                     type = "number[]";
-                    decorators.push("@Transform(({ value }) => value.map(id) => parseInt(id))");
+                    decorators.push("@Transform(({ value }) => value.map((id: string) => parseInt(id)))");
```

Comment:
@nsams this looks like a syntax error to me. I tried to fix it, but please verify if my change is needed.

Reply:
Yes, this is a syntax error. But I don't think we need the explicit type annotation.

```suggestion
                    decorators.push("@Transform(({ value }) => value.map((id) => parseInt(id)))");
```